client/src/components/Login.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState } from 'react'
+import {
+    withStyles, InputLabel, Link, Typography, DialogTitle, DialogContentText,
+    DialogContent, DialogActions, Dialog, TextField, Button, Snackbar
+} from '@material-ui/core';
+import MuiAlert from '@material-ui/lab/Alert';
+import axios from 'axios';
+
+function Alert(props) {
+    return <MuiAlert elevation={6} variant="filled" {...props} />;
+}
+const useStyles = theme => ({
+    form: {
+        display: 'flex',
+        flexDirection: 'column',
+        margin: 'auto',
+        // width: 'fit-content',
+        width: '50vw'
+    },
+    title: {
+        textAlign: 'center',
+        fontWeight: '1000',
+
+    },
+    inputLabel: {
+        marginTop: theme.spacing(3),
+        fontWeight: '900',
+        fontSize: '12px',
+    },
+
+})
+function Login(props) {
+    const [open, setOpen] = useState(false);
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [emailErr, setEmailErr] = useState('');
+    const [passwordErr, setPasswordErr] = useState('');
+    const [loginErr, setLoginErr] = useState(false);
+    const onLoginSubmit = () => {
+        const isValid = validate();
+        if (isValid) {
+            axios.post('users/login', {
+                userEmail: email,
+                password: password
+            })
+                .then(response => {
+                    if (response.status === 200) {
+                        console.log("success")
+                        // update App.js state
+                        props.updateUser({
+                            loggedIn: true,
+                            username: response.data.username
+                        })
+                        // clear form and set the sucessful switch to true
+                        setEmail('');
+                        setPassword('');
+                        setEmailErr('');
+                        setPasswordErr('');
+                    }
+                }).catch(error => {
+                    setLoginErr(true);
+                })
+        }
+    }
+
+    const handleCloseSnackbar = (event, reason) => {
+        if (reason === 'clickaway') {
+            return;
+        }
+        setLoginErr(false);
+    };
+
+    const validate = () => {
+        let emailErr = "";
+        let passwordErr = "";
+
+        if (!email.includes("@")) {
```

Comment:
Not a big deal but maybe using something like the [answer here](https://stackoverflow.com/questions/46155/how-to-validate-an-email-address-in-javascript) for email validation?